iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepKeyword.java

```diff
@@ -38,7 +38,10 @@ public enum BicepKeyword implements GrammarRuleKey {
   IMPORT("import"),
   WITH("with"),
   AS("as"),
-  MODULE("module");
+  MODULE("module"),
+  TRUE("true"),
+  FALSE("false"),
+  NULL("null");
```

Comment:
Is it used? I don't see it in this PR, we have TRUE_LITERAL_VALUE and similar to detect `literalValue`

Reply:
It is not used directly, but by adding them in `BicepKeyword`, they get injected into the regex exception of `Identifier`, so that we avoid having them be considered as valid `Identifier` instead of a literal value.
Although, now that I see it, I don't think `true`/`false`/`null` follow the definition of a keyword.
I will remove them from the list and instead use those literals in the regex build method.